source/fundamentals/data-formats/documents.txt

```diff
@@ -284,6 +289,94 @@ For more information on retrieving and manipulating MongoDB data, see our
 For more information on this class, see the
 :java-docs:`BsonDocument API documentation </apidocs/bson/org/bson/BsonDocument.html>`.
 
+JsonObject
+-------------
+
+The ``JsonObject`` class acts as a wrapper for JSON strings.
+``JsonObject`` is a more performant alternative to both ``Document``
+and ``BsonDocument`` when working with JSON strings because it does not create an intermediate
+``Map`` object.
+
+By default, ``JsonObject`` stores :manual:`Extended JSON </reference/mongodb-extended-json/>`.
+You can customize the format of JSON in ``JsonObject`` by specifying a
+:java-docs:`JsonObjectCodec</apidocs/bson/org/bson/codecs/JsonObjectCodec.html>`
+and passing it a
+:java-docs:`JsonWriterSettings</apidocs/bson/org/bson/json/JsonWriterSettings.html>`
+object. For more information on JSON formats, see
+our :doc:`Extended JSON guide </fundamentals/data-formats/document-data-format-extended-json/>`.
+For more information on specifying codecs, see our 
+:doc:`Codecs guide </fundamentals/data-formats/codecs>`.
+
+In the following code snippet, we show how to instantiate a sample ``JsonObject`` 
+instance wrapping an Extended JSON string containing different types of key value pairs:
+
+.. code-block:: java
+
+   String ejsonStr =  "{"
+                     +" \"_id\": {"
+                     +"  \"$oid\": \"6035210f35bd203721c3eab8\""
+                     +" },"
+                     +" \"name\": \"Gabriel Garc\\u00eda M\\u00e1rquez\","
+                     +" \"dateOfDeath\": {"
+                     +"  \"$date\": \"2014-04-17T04:00:00Z\""
+                     +" },"
+                     +" \"novels\": ["
+                     +"  {"
+                     +"   \"title\": \"One Hundred Years of Solitude\","
+                     +"   \"yearPublished\": 1967"
+                     +"  },"
+                     +"  {"
+                     +"   \"title\": \"Chronicle of a Death Foretold\","
+                     +"   \"yearPublished\": 1981"
+                     +"  },"
+                     +"  {"
+                     +"   \"title\": \"Love in the Time of Cholera\","
+                     +"   \"yearPublished\": 1985"
+                     +"  }"
+                     +" ]"
+                     +"}";
+   JsonObject author = new JsonObject(ejsonStr);
+
+To insert this document into a collection, instantiate a collection using
+:java-docs:`getCollection()</apidocs/mongodb-driver-sync/com/mongodb/client/MongoDatabase.html#getCollection(java.lang.String,java.lang.Class)>`,
```

Comment:
While the snooty parser usually can parse this, make sure to separate by a space as the part before the angled braces could contain an angled brace itself.
```suggestion
:java-docs:`getCollection() </apidocs/mongodb-driver-sync/com/mongodb/client/MongoDatabase.html#getCollection(java.lang.String,java.lang.Class)>`,
```

Reply:
Thanks for the note. I am committing your suggestion.

Takeaway:

Always separate link text from ling angle bracket to avoid possible issue where link text angle bracket is mistaken for angle bracket signaling start of link.